django_cte/meta.py

```diff
@@ -93,6 +93,15 @@ def relabeled_clone(self, relabels):
             clone._alias = relabels[self._alias]
         return clone
 
+    def resolve_expression(self, query=None, allow_joins=True, reuse=None,
```

Comment:
It would make a nicer overall diff (easier to review the entire PR) if this was moved above `relabeled_clone` where it was before.

Reply:
That's the plan. I was able to get commcare-hq to build and run the tests. I pushed this to run those tests locally. Once I can confirm it's working properly, I'll clean the PR up and let you know.